config/core/test-configs.yaml

```diff
@@ -1906,6 +1912,7 @@ test_configs:
       - baseline
       - baseline-nfs
       - sleep
+      - smc
```

Comment:
Which lab has this board? There's no result on staging.

Reply:
@aliceinwire How about this one?  Have there been any results, and do you know which labs have this device type?

Reply:
I have no idea either. is this not the imx device ? https://staging.kernelci.org/soc/imx/job/cip/

Reply:
Found it in lab-pengutronix:
https://hekla.openlab.pengutronix.de/scheduler/device_type/imx23-olinuxino

But it's been offline for months.  So better not pick this one.

If you want some i.MX platform to be covered, maybe pick `imx6q-sabrelite` as that's well supported.  There are several online in Collabora's lab (I think we should soon have 10 online).